test/app-swift-kitura-test.js

```diff
@@ -258,14 +258,26 @@ function testServiceDependencies(serviceName, dependencies) {
 }
 
 function testServiceInstrumentation(serviceName, servLookupKey, codeForServices) {
+	let serviceVariable = {
+		"service-alert-notification": "alertNotificationService",
+		"service-appid": "appidService",
+		"service-cloudant": "couchDBService",
+		"service-object-storage": "objectStorageService",
+		"service-redis": "redisService",
+		"service-mongodb": "mongoDBService",
+		"service-postgre": "postgreSQLService",
+		"service-push": "pushNotificationService",
+		"service-watson-conversation": "watsonConversationService"
+	}
 	function pascalize(name) {
 		return name.split('-').map(part => part.charAt(0).toUpperCase() + part.substring(1).toLowerCase()).join('');
 	}
-	let expectedInitFunction = `initialize${pascalize(serviceName)}()`;
-	yassert(codeForServices.indexOf(`try ${expectedInitFunction}`) !== -1);
+	let expectedInitFunctionCall = `initialize${pascalize(serviceName)}(cloudEnv: cloudEnv)`;
+	let expectedInitFunctionTemplate = `initialize${pascalize(serviceName)}(cloudEnv: CloudEnv)`;
```

Comment:
This confused me for a second because of the variable name (and the similarity of the value to `expectedInitFunctionCall`. Perhaps `expectedInitFunctionDeclaration` would be a clearer name?

Reply:
done